### PR TITLE
simplify observability diagrams

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- move **Access+Expose Kiali, Grafana, Jaeger** to Security section
- create new diagram for accessing Kiali, Grafana, Jaeger
- simplify other observability diagrams and refer to **Access+Expose Kiali, Grafana, Jaeger** for details
- add link to tracing flow and rephrase sentence in **Detault Istio Setup** 
- remove OpenTelemetry from main areas overview diagram

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/11889

### CODE REVIEW

File: docs/01-overview/main-areas/service-mesh/smsh-02-default-istio-setup-in-kyma.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ These configuration changes are applied to customize Istio for use with Kyma:
 - Automatic sidecar injection is enabled by default, excluding the `istio-system` and `kube-system` Namespaces. See how to [disable sidecar proxy injection](../../../04-operation-guides/operations/smsh-01-istio-disable-sidecar-injection.md).
 - Resource requests and limits for Istio sidecars are modified to best suit the needs of the evaluation and production profiles.
 - [Mutual TLS (mTLS)](https://istio.io/docs/concepts/security/#mutual-tls-authentication) is enabled cluster-wide in a STRICT mode.
-- Global tracing is set to use the Zipkin installation provided by Kyma.
+- Global tracing is set to use the Zipkin protocol to send requests to the tracing component provided by Kyma. Learn more about [traciing]](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
```
Comment (a-thaler) [inline docs/01-overview/main-areas/service-mesh/smsh-02-default-istio-setup-in-kyma.md:29]:
```suggestion
- Global tracing is set to use the Zipkin protocol to send requests to the tracing component provided by Kyma. Learn more about [tracing]](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
```

File: docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md [hunk 2]
```diff
@@ -23,7 +23,5 @@ The process of collecting traces by Jaeger looks as follows:
 The process of querying traces from Jaeger looks as follows:
 
 1. A Kyma user accesses the Jaeger UI to look for specific traces.
-2. The user has [two options](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information:
```
Comment (a-thaler) [inline docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md:26]: Can't we move that description to the "Access" document where the new diagram is to have a bit more description?

File: docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md [hunk 3]
```diff
@@ -6,10 +6,9 @@ The following diagram presents the overall Kiali architecture and the way the co
...
-3. If not, the Keycloak Gatekeeper redirects you to the identity provider to log in.
-4. After a successful log in, you can access the Kiali service which serves the website. The service exposes an endpoint and acts as an entry point for the Kiali deployment.
-5. Kiali collects the information on the cluster health from the following sources:
-   * API server which provides data on the cluster state.
-   * Service Mesh by analyzing metrics Prometheus scrapes from the Istio Pod.
+1. Use the Kyma Console or direct URL to access Kiali. Learn more about [accessing Kiali](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).
+2. Kiali collects the information on the cluster health from the following sources:
+   * Jaeger tracing
+   * Kubernetes API server, which provides data on the cluster state
+   * Service Mesh using Prometheus
```
Comment (a-thaler) [inline docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md:13]: Please move the last item to the first point in the list as it is most important.

Maybe it is worth to say that it is the metrics data coming from istio collected by prometheus

File: docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md [hunk 4]
```diff
@@ -6,10 +6,9 @@ The following diagram presents the overall Kiali architecture and the way the co
...
-1. Use the Kyma Console or direct URL to access Kiali.
-2. To ensure authentication, the Keycloak Gatekeeper checks if you have a valid token.
-3. If not, the Keycloak Gatekeeper redirects you to the identity provider to log in.
-4. After a successful log in, you can access the Kiali service which serves the website. The service exposes an endpoint and acts as an entry point for the Kiali deployment.
-5. Kiali collects the information on the cluster health from the following sources:
-   * API server which provides data on the cluster state.
-   * Service Mesh by analyzing metrics Prometheus scrapes from the Istio Pod.
+1. Use the Kyma Console or direct URL to access Kiali. Learn more about [accessing Kiali](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).
+2. Kiali collects the information on the cluster health from the following sources:
+   * Jaeger tracing
```
Comment (a-thaler) [inline docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md:11]: Trace data collected by Jaeger

**Review by a-thaler — CHANGES_REQUESTED** [review]

File: docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md (hunk 2 above)
Comment (NHingerl) [reply docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md:26]: as below, the current sentence is

`The user passes the request to the Jaeger Query service and retrieves the trace information. Learn more about [accessing Jaeger](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).`

Is that okay?

File: docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md (hunk 4 above)
Comment (NHingerl) [reply docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md:11]:
```suggestion
   * Trace data collected by Jaeger
```

File: docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md (hunk 3 above)
Comment (NHingerl) [reply docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md]: done with new commit

**Review by a-thaler — APPROVED** [review]

**Pull request merged**